pkg/build/nodeimage/internal/container/docker/archive.go

```diff
@@ -26,15 +26,21 @@ import (
 	"os"
 	"strings"
 
+	"github.com/opencontainers/go-digest"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"sigs.k8s.io/kind/pkg/errors"
 )
 
+// ioContainerdImageName appears as an annotation in a manifest entry in "index.json".
+const ioContainerdImageName = "io.containerd.image.name"
+
 // GetArchiveTags obtains a list of "repo:tag" docker image tags from a
 // given docker image archive (tarball) path
 // compatible with all known specs:
 // https://github.com/moby/moby/blob/master/image/spec/v1.md
 // https://github.com/moby/moby/blob/master/image/spec/v1.1.md
 // https://github.com/moby/moby/blob/master/image/spec/v1.2.md
+// https://github.com/opencontainers/image-spec/blob/v1.0.2/image-index.md  (annotation "io.containerd.image.name")
```

Comment:
this link does not reference the containerd annotation

Reply:
Removed the link